packages/base/src/CustomElementsRegistry.js

```diff
@@ -0,0 +1,23 @@
+const DefinitionsSet = new Set();
```

Comment:
The purpose of this module is to avoid a cyclic dependency b/n RenderScheduler and UI5Element